### PR TITLE
Bug 864617 - updated all apps and removed soundcloud

### PR DESCRIPTION
**Pull request by yurenju**

### CODE REVIEW

File: external-apps/facebook/metadata.json [hunk 1]
```diff
@@ -1 +1 @@
-{"origin": "http://m.facebook.com/openwebapp/"}
\ No newline at end of file
+{"origin": "http://m.facebook.com/"}
```
Comment (fabricedesre) [inline external-apps/facebook/metadata.json]: You should remove the trailing / from the origin.
Why is there no installOrigin and etag? This looks incomplete

File: external-apps/heremaps/metadata.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"origin": "app://heremaps/", "installOrigin": "https://marketplace.firefox.com", "packageEtag": "\"e1739849fee7f0377677fe9b537b0f47afa412eef28cc4b60510084a2973ac44\"", "manifestURL": "https://marketplace.firefox.com/app/7eccfd71-2765-458d-983f-078580b46a11/manifest.webapp", "etag": "\"3929b97863ce806ba5aec2417167388b\"", "type": "web"}
```
Comment (fabricedesre) [inline external-apps/heremaps/metadata.json]: No trailing / for the origin please.

File: external-apps/notes/metadata.json [hunk 3]
```diff
@@ -1 +1 @@
-{"origin": "http://everythingme.github.com/ffos-notes/"}
\ No newline at end of file
+{"origin": "http://everythingme.github.com/"}
```
Comment (fabricedesre) [inline external-apps/notes/metadata.json]: same here

File: external-apps/twitter/metadata.json [hunk 4]
```diff
@@ -1 +1 @@
-{"origin": "https://mobile.twitter.com/"}
+{"origin": "https://mobile.twitter.com/"}
```
Comment (fabricedesre) [inline external-apps/twitter/metadata.json]: and there

File: external-apps/wikipedia/metadata.json [hunk 5]
```diff
@@ -1 +1 @@
-{"origin": "https://bits.wikimedia.org/WikipediaMobileFirefoxOS/"}
\ No newline at end of file
+{"origin": "https://bits.wikimedia.org/"}
```
Comment (fabricedesre) [inline external-apps/wikipedia/metadata.json]: and there

File: external-apps/youtube/metadata.json [hunk 6]
```diff
@@ -1 +1 @@
-{"origin": "app://youtube/", "installOrigin": "https://marketplace.firefox.com", "packageEtag": "59b70b6c4f18175bb19c1c25c6c536fdf49d177f00b0d720ba68dedadf842d88", "manifestURL": "https://marketplace.firefox.com/app/9a46c4e6-0f8b-40cb-a770-11ea1b7fda20/manifest.webapp", "etag": "c0216f1ee14450d5af4ae5c90e25bfee", "type": "web"}
\ No newline at end of file
+{"origin": "http://m.youtube.com/"}
```
Comment (fabricedesre) [inline external-apps/youtube/metadata.json]: and there

File: external-apps/facebook/metadata.json (hunk 1 above)
Comment (yurenju) [inline external-apps/facebook/metadata.json]: do we need installOrigin & etag both for packaged & hosted app?

**Pull request merged**